interfaces/console/ui/molecules/SubscriberDetails/index.tsx

```diff
@@ -0,0 +1,457 @@
+    setSimAction(event.currentTarget);
+  };
+  const handleCloseSimAction = () => {
+    setSimAction(null);
+  };
+  const handleSimMenu = (action: string, simId: string) => {
+    handleCloseSimAction();
+
+    handleSimActionOption(action, simId, subscriberInfo.uuid);
+  };
+  useEffect(() => {
+    if (subscriberInfo) {
+      setSubscriberLoading(false);
+      setEmail(subscriberInfo.email);
+      setFistName(subscriberInfo.firstName);
+    }
+  }, [subscriberInfo]);
+
+  const handleSimEdit = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+  const handleEditName = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setFistName(event.target.value);
+  };
+  const handleSaveSubscriber = () => {
+    setOnEditEmail(false);
+    setOnEditName(false);
+    handleUpdateSubscriber(subscriberInfo.uuid, email, fistName);
+  };
+
+  return (
+    <Dialog
+      open={ishowSubscriberDetails}
+      onClose={handleClose}
+      maxWidth="sm"
+      fullWidth
+    >
+      <DialogTitle id="alert-dialog-title">
+        <Stack direction="row" spacing={2} alignItems={'center'}>
+          <Typography variant="h6">
+            {subscriberInfo && subscriberInfo.firstName}
+          </Typography>
+          <IconButton
+            aria-controls="menu"
+            aria-haspopup="true"
+            onClick={handleClick}
+          >
+            <MoreVertIcon />
+          </IconButton>
+          <Menu
+            id="menu"
+            anchorEl={anchorEl}
+            open={Boolean(anchorEl)}
+            onClose={handleCloseItem}
+          >
+            <MenuItem
+              onClick={() =>
+                handleMenuItemClick('pauseService', subscriberInfo.uuid)
+              }
+            >
+              Pause service
+            </MenuItem>
+            <MenuItem
+              onClick={() =>
+                handleMenuItemClick('deleteSubscriber', subscriberInfo.uuid)
+              }
+              sx={{ color: colors.red }}
+            >
+              Delete subscriber
+            </MenuItem>
+          </Menu>
+        </Stack>
+      </DialogTitle>
+      <IconButton
+        aria-label="close"
+        onClick={handleClose}
+        sx={{
+          position: 'absolute',
+          right: 8,
+          top: 8,
+        }}
+      >
+        <CloseIcon />
+      </IconButton>
+      <Box sx={{ width: '100%' }}>
+        <Box sx={{ p: 2 }}>
+          <Tabs
+            value={selectedTab}
+            onChange={handleTabChange}
+            aria-label="tabs menu"
+          >
+            <Tab label="Information" />
+            <Tab label="Data Usage" />
+            <Tab label="SIMs" />
+            <Tab label="History" />
+          </Tabs>
+        </Box>
+
+        <Box sx={{ pl: 2 }}>
+          <Typography
+            component="div"
+            role="tabpanel"
+            hidden={selectedTab !== 0}
+          >
```

Comment:
We can use `Box` here, rather `Typography`

Reply:
Addressed